lib/flipper/api/error_response.rb

```diff
@@ -24,6 +24,7 @@ def as_json
         feature_not_found: Error.new(1, "Feature not found.", "", 404),
         group_not_registered: Error.new(2, "Group not registered.", "", 404),
         percentage_invalid: Error.new(3, "Percentage must be a positive number less than or equal to 100.", "", 400),
+        flipper_id_invalid: Error.new(4, "Required parameter flipper_id cannot be nil.", "", 400),
```

Comment:
yes, I've gone back and forth on this. I was thinking that saying its missing sounds like the key is missing, but definitely see how nil doesn't really translate to JSON.  I like your suggestion as it being missing could also be interpreted as the key or value is missing. Will update thanks!